Dockerfile_tesseract_training

```diff
@@ -1,31 +1,33 @@
 
-# Run this dockerfile using the command below from repo root
-# docker-compose -f docker.dev.yml up -d
+# Run this dockerfile using the command below from repo root for model training
```

Comment:
Since this new Dockerfile combines the functionality of training and running the API, you should rename this file to `Dockerfile` and delete the other one. This will require changes to the docker compose file.